examples/add_volume_series.py

```diff
@@ -0,0 +1,18 @@
+"""
+Display one 3-D volume layer using the add_volume API
+"""
+
+from skimage import data
+import napari
+
+
+with napari.gui_qt():
+    blobs = data.binary_blobs(length=128, blob_size_fraction=0.05, n_dim=4)
+    blobs = blobs.astype(float) * 255
+    viewer = napari.Viewer()
+    for i in range(blobs.shape[0]):
```

Comment:
you should remove the for loop and just make one call 
```python
layer = viewer.add_volume(blobs)
```
This should then generate a single layer that has a slider at the bottom of the screen. Using the slider should move you through the data

Reply:
volume node that is initialized when volume layer is called can only take 3D data. If I remove the loop it would fail. 

``` python ----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/Users/pranathivemuri/czbiohub/napari/napari/layers/volume/tests/test_volume.py", line 59, in test_4D_volume
    layer = Volume(data)
  File "/Users/pranathivemuri/czbiohub/napari/napari/layers/volume/volume.py", line 83, in __init__
    visual = VolumeNode(volume, threshold=0.225, emulate_texture=False)
  File "/usr/local/lib/python3.6/site-packages/vispy/scene/visuals.py", line 127, in __init__
    subclass.__init__(self, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/vispy/visuals/volume.py", line 443, in __init__
    self.set_data(vol, clim)
  File "/usr/local/lib/python3.6/site-packages/vispy/visuals/volume.py", line 465, in set_data
    raise ValueError('Volume visual needs a 3D image.')
ValueError: Volume visual needs a 3D image.```

Reply:
hmm ok I can look at this, the slicing should have fixed this now